test/integration/user_authorization_test.rb

```diff
@@ -14,20 +14,24 @@ def setup
                         visit root_url
   end
 
-  test "an unauthorized user cannot see another user's order history" do
-    user1 = User.create(username: 'user',
-                        password: 'password',
-                        first_name: 'John',
-                        last_name: 'Doe',
+  test "an unauthorized user cannot see another user's order history and only his own" do
+    user1 = User.create(username: 'xxx',
+                        password: 'yyy',
+                        first_name: 'Jeff',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.